src/unicef_security/graph.py

```diff
@@ -23,7 +23,9 @@
 
 DJANGOUSERMAP = {'_pk': ['username'],
                  'username': 'userPrincipalName',
-                 'email': 'mail',
+                 # Looks like sometimes email is kept in `userPrincipalName` and `mail` is empty
```

Comment:
@saxix can you review this?

Reply:
I don't know what's happening in Azure AD that my user has this field as `Null`, and i can't set it anywhere. I have checked other users too, and the `email` field seems to be kept in `userPrincipalName`. 
After/If this change is rolled back, the cassettes have to be regenerated with an Azure AD user that has the `mail` field set. After that most likely few of the test have to be adjusted to match the new data in the cassettes.